modules/hostap/src/supp_api.c

```diff
@@ -41,10 +41,12 @@ enum status_thread_state {
 
 #define DISCONNECT_TIMEOUT_MS 5000
 
-#define _wpa_cli_cmd_v(cmd, ...) \
+#define _wpa_cli_cmd_v(cmd, status, ...) \
 	do { \
-		if (z_wpa_cli_cmd_v(cmd, ##__VA_ARGS__) < 0) { \
+		int ec = z_wpa_cli_cmd_v(cmd, ##__VA_ARGS__); \
```

Comment:
You can directly use `ret` here, as this is a macro.

Reply:
Hmm sure, but the supp_shell_connect_status uses different naming (`status` instead of `ret`), so I wanted to keep the macro generic.

Reply:
We can change that, in general the conditional macros rely on a common variable for error checking and don't define it themself, makes the code readable.

Reply:
Ok, will fix that, thanks.